Render/MapRendering/DungeonRenderer.js

```diff
@@ -78,7 +78,9 @@ class DungeonRenderer extends MapTab {
             renderLibs.stopScizzor()
         }
 
-        if (true) {
+        if (!renderContext.image
+            || (renderContext.imageLastUpdate < dungeonMap.lastChanged)
+            || renderContext.settings.spinnyMap) {
```

Comment:
wait why does the map need to render every frame if spinny map is being used?

Reply:
It's to hurt the nons!

No, it's actually cause I'm an idiot, I'm gonna fix that.

For some reason, I thought rotating the renderer didn't rotate the image but it does...